### PR TITLE
feat(@desktop/profile): Adapt view for settings as pre new design

### LINKED ISSUE #5725: adapt view for settings as shown during account generation
Currently in wallet we segregate the accounts derived from default account and imported seed phrase.

it looks like this https://www.figma.com/file/FkFClTCYKf83RJWoifWgoX/Wallet-v2?node-id=5248%3A188675\

However it should look something like this https://www.figma.com/file/idUoxN7OIW2Jpp3PMJ1Rl8/%E2%9A%99%EF%B8%8F-Settings-%7C-Desktop?node-id=1869%3A134081

### PR DESCRIPTION
**Pull request by Khushboo-dev-cpp**
fixes #5725

### What does the PR do

Adapt settings profile screen to show related accounts in accounts view 

https://www.figma.com/file/G1GVE21KLEWwBwT1uK6IIh/Related-accounts?node-id=6935%3A174467

### Affected areas

Profile

### Screenshot of functionality (including design for comparison)

- [x] I've checked the design and this PR matches it

https://user-images.githubusercontent.com/60327365/179766683-f72c6176-4040-46dd-91f7-e20bdef32b68.mov

### CODE REVIEW

File: src/app/modules/main/wallet_section/accounts/compact_item.nim [hunk 1]
```diff
@@ -0,0 +1,89 @@
+import strformat
+
+type
+  Item* = object
```
Comment (alaibe) [inline src/app/modules/main/wallet_section/accounts/compact_item.nim:4]: don't we already have a similar item and model?

**Review by alaibe — APPROVED** [review]
:100:

File: ui/app/AppLayouts/Profile/controls/InformationTile.qml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    height: 52
+    width: layout.width + Style.current.xlPadding
+    radius: Style.current.radius
+    border.width: 1
+    border.color: Theme.palette.baseColor2
+    color: Style.current.transparent
+
+    ColumnLayout {
+        id: layout
+        spacing: -1
```
Comment (caybro) [inline ui/app/AppLayouts/Profile/controls/InformationTile.qml:27]: Why negative spacing?

File: ui/app/AppLayouts/Profile/controls/InformationTile.qml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+            id: primaryText
+            font.pixelSize: 13
+            color: Theme.palette.directColor5
+            visible: text
+        }
+        StatusBaseText {
+            id: secondaryText
+            font.pixelSize: 15
+            color: Theme.palette.directColor1
+            visible: text
```
Comment (caybro) [inline ui/app/AppLayouts/Profile/controls/InformationTile.qml:39]: You should set width (fillWidth) for those texts, and enable `elide` for the case when the text is too long

File: ui/app/AppLayouts/Profile/controls/InformationTile.qml [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+Rectangle {
+    id: root
+
+    property alias primaryText: primaryText.text
+    property alias secondaryText: secondaryText.text
+    property alias tagsModel: tags.model
+    property alias tagsDelegate: tags.delegate
+
+    height: 52
+    width: layout.width + Style.current.xlPadding
```
Comment (caybro) [inline ui/app/AppLayouts/Profile/controls/InformationTile.qml:19]: This should be `implicitWidth` (as designed) and the layout should just fill `root`; same for height (make it `implicitHeight`)

**Review by caybro — CHANGES_REQUESTED** [review]
Just some minor stuff on the QML side, otherwise looks nice! :)

File: src/app/modules/main/wallet_section/accounts/compact_item.nim (hunk 1 above)
Comment (Khushboo-dev-cpp) [reply src/app/modules/main/wallet_section/accounts/compact_item.nim:4]: Hey @alaibe yes we do ! this is like just a reduced version of the account `item.nim` and `model.nim`. I need this list of `related accounts within an account item` and it was simpler to just create this.

File: ui/app/AppLayouts/Profile/controls/InformationTile.qml (hunk 2 above)
Comment (Khushboo-dev-cpp) [reply ui/app/AppLayouts/Profile/controls/InformationTile.qml:27]: without it the spacing is higher than we see in the design
image

**Review by caybro — APPROVED** [review]

**Review by iurimatias — APPROVED** [review]

**Pull request merged**